packages/react/src/components/Search/Search.tsx

```diff
@@ -257,6 +256,7 @@ const Search = React.forwardRef<HTMLInputElement, SearchProps>(function Search(
         type={type}
         value={value}
         tabIndex={onExpand && !isExpanded ? -1 : undefined}
+        {...rest}
```

Comment:
I'm not sure why we had this placed before all the existing props. Props passed by consumers would always be overwritten by what we had here.

It's possible this could cause some unintended effects downstream, but I don't think it's necessarily a breaking change. Thoughts?

Reply:
Is the only "unintended effect" that props users were passing and expecting to work will now actually work? 😅

Reply:
I don't think this is breaking either, more like a bug fix for something that should have been working.